cmd/gen/makefile/template/template.go

```diff
@@ -0,0 +1,67 @@
+package template
+
+//TemplateConfig inherits all placeholder which should be overwritten.
+type TemplateConfig struct {
+	Application string
+}
+
+var Makefile = `APPLICATION   ?= {{.Application}}
+VERSION       ?= $(shell git describe --tags --always --dirty)
+SOURCES       = $(shell find . -name '*.go')
+GOPKGS        = $(shell go list ./...)
+TAG           ?= $(VERSION)
+BUILD_FLAGS   ?= -v
```

Comment:
It would be good to make it on par with the [orb](https://github.com/giantswarm/architect-orb/blob/7e3e1243d49a67d6f07482e40145e3da3598ded2/src/commands/go-test.yaml#L5-L9). But it's fine with me if we don't do this in first iteration.